xboxlib.c

```diff
@@ -204,6 +205,31 @@ void gigimport(giant g, unsigned char *buff,int len){
 	
 
 }
+#endif
+
+void gigimport(giant g, unsigned char *buff, int len) {
+
+	// copy buffered 'number' into giant's number buffer
+	//int count;
+	memcpy(g->n, buff, len);
+
+	// the giants number buffer stores large numbers as
+	// multiple 16bit numbers, so if the length of our buffer is odd
+	// our number wont fit properly.
+	// to fix this we add a trailing 0x00 byte
+	if (len & 1)
+	{
+		*((unsigned char*)(g->n) + len) = 0x00;
+		len++;
+	}
+	g->sign = len / 2;
+
+	if (g->sign == 0)
+	{
```

Comment:
nit: the rest of the code seems to have `{` on the same line as the `if`. Other instance above affected too.